playground/playground.html

```diff
@@ -5,8 +5,8 @@
   <title>LUCUMA-SSO LOCAL PLAYGROUND</title>
   <script>
 
-    var ssoRootUri     = "https://sso.gpp.lucuma.xyz"
-    // var ssoRootUri     = "http://local.lucuma.xyz:8080"
+    // var ssoRootUri     = "https://sso.gpp.lucuma.xyz"
+    var ssoRootUri     = "http://local.lucuma.xyz:8080"
```

Comment:
Did you mean to keep this pointing to local?

Reply:
Yeah this is just a little web page you can run locally to test out SSO, it should be pointed to local by default.